packages/react-native-reanimated/src/mutables.ts

```diff
@@ -69,12 +90,14 @@ function makeMutableNative<Value>(initial: Value): Mutable<Value> {
 
   const mutable: Mutable<Value> = {
     get value(): Value {
+      checkInvalidReadDuringRender();
```

Comment:
Do we want to check it always or only in the Development mode?

Reply:
Yea, makes sense to limit only to development. I will change that.